### PR TITLE
Feat/sva repayment 2

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/sva`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7121
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

New global Telecom / Telephony / Repayments page

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.routing.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        },
+      },
+      resolve: {
+        breadcrumb: /* @ngInject */ ($translate) =>
+          $translate.instant('telephony_repayments_title'),
+        svaWalletLink: /* @ngInject */ (coreURLBuilder) =>
+          coreURLBuilder.buildURL('telecom', '#/telephony/sva-wallet'),
+        hasSvaWallet: /* @ngInject */ ($q, $state, svaWallet) => {
+          if (svaWallet && svaWallet.status === 'VALID') return true;
+          return $q.reject().catch(() => {
```
Comment (frenautvh) [inline packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.routing.js:27]: looks the same as just returning :
```js
$state.go('telecom.telephony');
```

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.routing.js (hunk 1 above)
Comment (y4nnL) [reply packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.routing.js]: :+1:  I though state redirections were prohibited inside router resolutions

**Review by zakariasahmane — DISMISSED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
@zakariasahmane Could you please resolve existing conflicts?
Thank you

**Review by antleblanc — CHANGES_REQUESTED** [review]
@y4nnL Could you please resolve existing conflicts?
Thank you!

**Comment by y4nnL** [comment]
Closing this until the specifications are finalized

**Pull request closed without merging**